### PR TITLE
fix(Message): optimize getters and provided props

### PR DESCRIPTION
**Pull request by Antreesy**
### ☑️ Resolves

* Delete unused `scrollerBoundingClientRect` prop
* Move computation of `isLastCallStartedMessage` to store
* Make update of `hasCall` more clear across the app
  * Prevent notifications from updating store if already done
### 🖼️ Screenshots / Screencasts

No visual changes

### 🏁 Checklist

- [x] 🌏 Tested with Chrome, Firefox and Safari or should not be risky to browser differences
- [x] ⛑️ Tests are included or not possible

### CODE REVIEW

File: src/components/MessagesList/MessagesGroup/Message/Message.vue [hunk 1]
```diff
@@ -508,23 +503,12 @@ export default {
...
-			return this.$store.getters.messagesList(this.token)
-		},
-
 		isLastCallStartedMessage() {
-			// FIXME: remove dependency to messages list and convert to property
-			const messages = this.messagesList
-			// FIXME: don't reverse the whole array as it would create a copy, just do an actual reverse search
-			const lastCallStartedMessage = messages.reverse().find((message) => message.systemMessage === 'call_started')
-			return lastCallStartedMessage ? (this.id === lastCallStartedMessage.id) : false
+			return this.systemMessage === 'call_started' && this.id === this.$store.getters.getLastCallStartedMessageId(this.token)
```
Comment (DorraJaouad) [inline src/components/MessagesList/MessagesGroup/Message/Message.vue:507]: You can return the value of the getter and use `getToken` in the store

File: src/store/conversationsStore.js [hunk 2]
```diff
@@ -216,11 +216,7 @@ const mutations = {
 	},
 
 	overwriteHasCallByChat(state, { token, hasCall }) {
-		if (hasCall) {
-			Vue.set(state.conversations[token], 'hasCallOverwrittenByChat', hasCall)
-		} else {
-			Vue.delete(state.conversations[token], 'hasCallOverwrittenByChat')
-		}
+		Vue.set(state.conversations[token], 'hasCallOverwrittenByChat', hasCall)
 	},
```
Comment (DorraJaouad) [inline src/store/conversationsStore.js:220]: So why are you keeping it then ?

Comment (Antreesy) [reply src/store/conversationsStore.js:220]: good question. I thought to mark the conversation as modified, until we fetch a proper one from server

**Review by DorraJaouad — APPROVED** [review]
Tested 🟢

**Pull request merged**